src/gstreamer/launch/laptop_launch.py

```diff
@@ -30,6 +27,7 @@ def generate_launch_description():
         cmd=["rqt", "--force-discover", "--standalone", "CameraClient"], shell=True, output="screen"
     )
 
+    # NOTE: RVIZ uses a lot of bandwidth, so it should be turned off during competition matches
```

Comment:
Why was this added in this pr

Reply:
Simply because I forgot to add it earlier / when I was looking at the laptop_launch file I realized I should probably add some sort of note about rviz bandwidth in case we forget or something.